Products/ZenUI3/browser/resources/css/zen-cse.css

```diff
@@ -2943,3 +2990,44 @@ BYE BYE BORDERS
     .z-cse.z-cse-dark .edited_feedback .x-panel-header-text {
         color: yellow !important;
     }
```

Comment:
'yellow' is a bit too harsh.  Match the new flare background color #cdf856 instead.

Reply:
changed.